test/ApplicationFactory.ts

```diff
@@ -24,7 +24,7 @@ export const getDefaultConfig = () => {
     };
 };
 
-export const createApp = (session?: Session) => createAppConfigurable(container => {
+export const createApp = (session?: Session, configureContainerBindings: (container: Container) => void = () => { }) => createAppConfigurable(container => {
```

Comment:
This line feels a bit long, could we split into two lines for readability sake

Reply:
Shortened it.